mastering-plone/code.rst

```diff
@@ -184,4 +152,4 @@ The final result should look like this:
 .. figure:: ../_static/code_tree.png
    :align: center
 
-In earlier versions of the training we used a folder-based approach such as in https://github.com/collective/ploneconf.site_sneak. It proved to be a lot a lot of tedious work to maintain that.
+In earlier versions of the training we used a folder-based approach such as in https://github.com/collective/ploneconf.site_sneak. It proved to be a lot of tedious work to maintain that.
```

Comment:
I don't think this line helps current students; I would remove it, because it's only of historical interest (not that I have anything against history or historians!).